src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTableUnitTest.java

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import htsjdk.variant.variantcontext.*;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.testutils.VariantContextTestUtils;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.testng.Assert.*;
+
+/**
+ * Created by gauthier on 6/4/18.
+ */
+public class VariantsToTableUnitTest {
+    final Allele refAllele = Allele.create("A", true);
+    final Allele altAllele1 = Allele.create("T");
+    final Allele altAllele2 = Allele.create("C");
+
+    @Test
+    public void testExtractFields() throws Exception {
+        final VariantsToTable VTTtest = VariantsToTable.getInstance();
```

Comment:
I want to say you can replace this with `new VariantsToTable()`.

Reply:
But really the root of the problem is that `extractFields` is an instance method when it should be a static method that takes as parameters lists that are currently instance variables such as `genotypeFieldsToTake`